CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
+
+### Changes
+[BACKWARDS INCOMPATIBILITY NTOES]
+* Instead of sending `null` as default value for some tags we now send `unknown`
+* Event messages got changed.
+* Node tag is set by default
+
+[BUGFIX] Fix rate submission. See [179](https://github.com/DataDog/jenkins-datadog-plugin/pull/179) #TODO Confirm if the issue fixed in this PR was in the last release or not
+
+[IMPROVEMENT] Add note about how to send data to EU [154](ttps://github.com/DataDog/jenkins-datadog-plugin/pull/134)
+[IMPROVEMENT] Send delivery KPIs. See [132](https://github.com/DataDog/jenkins-datadog-plugin/pull/132) & [156](https://github.com/DataDog/jenkins-datadog-plugin/pull/156) (Thanks @pgarbe)
+[IMPROVEMENT] Add more granular statuses (i.e. `Not Built`, etc) to onCompleted event. See [153](https://github.com/DataDog/jenkins-datadog-plugin/pull/153) (Thanks @mbaitelman)
+[IMPROVEMENT] Lint project and remove unused code. See [160](https://github.com/DataDog/jenkins-datadog-plugin/pull/160)
+[IMPROVEMENT] Overall code refactor and cleanup. See [161](https://github.com/DataDog/jenkins-datadog-plugin/pull/161)
+[IMPROVEMENT] Add thread safety to dogstatsd submissions, adds `jenkins.job.started` and `jenkins.scm.checkout`, and add resiliency around potential build failures. See [169](https://github.com/DataDog/jenkins-datadog-plugin/pull/169)
+[IMPROVEMENT] Allow multiple values for tags & Support prefix for White/Blacklist & set Node tag by default & Added API + Target URL validations. See [172](https://github.com/DataDog/jenkins-datadog-plugin/pull/172)
+[IMPROVEMENT] Add 1 minute request timeout. See [174](https://github.com/DataDog/jenkins-datadog-plugin/pull/174) (Thanks @Mischa-Alff)
+[IMPROVEMENT] Add SVN tag support. See [175](https://github.com/DataDog/jenkins-datadog-plugin/pull/175)
+[IMPROVEMENT] Collect env variables and small code refactor. See [176](https://github.com/DataDog/jenkins-datadog-plugin/pull/176)
+[IMPROVEMENT] Refactor code that handles configs. See [177](https://github.com/DataDog/jenkins-datadog-plugin/pull/177)
+[IMPROVEMENT] Add executor, node, and queue metrics. Also adds total number of jobs metric. See [180](https://github.com/DataDog/jenkins-datadog-plugin/pull/180)
+[IMPROVEMENT] Add slave statistic metrics and more. Also adds security/SCM/system events. See [181](https://github.com/DataDog/jenkins-datadog-plugin/pull/181)
+[IMPROVEMENT] Refactor dogStatsDClient. See [183](https://github.com/DataDog/jenkins-datadog-plugin/pull/183)
+[IMPROVEMENT] Adding Global Tag file from workspace. See [182](https://github.com/DataDog/jenkins-datadog-plugin/pull/182)
+[IMPROVEMENT] Allow configuring plugin using env vars. See [184](https://github.com/DataDog/jenkins-datadog-plugin/pull/184)
+
+[OTHER] Update License and Github files. See [187](https://github.com/DataDog/jenkins-datadog-plugin/pull/187)
+[OTHER] Cleanup license and standard files. See [193](https://github.com/DataDog/jenkins-datadog-plugin/pull/193)
```

Comment:
Should it be part of the change log?

Reply:
I left it as other. It touches code that gets shipped, so my opinion is to include it 😄